task/06-conditions-n-loops-tasks.js

```diff
@@ -226,7 +267,14 @@ function reverseString(str) {
  *   34143 => 34143
  */
 function reverseInteger(num) {
-    throw new Error('Not implemented');
+    let reverse=[];
+    let buffer = num.toString();
+    for( let i = buffer.length -1; i>=0; i--)
+    {
+        reverse +=buffer[i];
+    }
+    return reverse;
+
 }
```

Comment:
Please, try to solve this task using Array method 'reverse'
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reverse
We can convert number  into string  at first, then transform it into array.